docs/source/developer/extension_tutorial.rst

```diff
@@ -130,45 +131,62 @@ You should see a list like the following.
 
 ::
 
-    LICENSE  README.md  package.json  src  style  tsconfig.json
+    LICENSE          MANIFEST.in      README.md        binder/          jupyterlab_apod/ package.json     pyproject.toml   setup.py         src/             style/           tsconfig.json
 
-Build and install the extension for development
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Commit what you have to git
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Your new extension project has enough code in it to see it working in
-your JupyterLab. Run the following commands to install the initial
-project dependencies and install it in the JupyterLab environment. We
-defer building since it will be built in the next step.
+Run the following commands in your ``jupyterlab_apod`` folder to
+initialize it as a git repository and commit the current code.
 
-.. note::
+.. code:: bash
 
-   This tutorial uses ``jlpm`` to install Javascript packages and
-   run build commands, which is JupyterLab's bundled
-   version of ``yarn``. If you prefer, you can use another Javascript
-   package manager like ``npm`` or ``yarn`` itself.
+    git init
+    git add .
+    git commit -m 'Seed apod project from cookiecutter'
+
+Note: This step is not technically necessary, but it is good practice to
+track changes in version control system in case you need to rollback to
+an earlier version or want to collaborate with others. For example, you
+can compare your work throughout this tutorial with the commits in a
+reference version of ``jupyterlab_apod`` on GitHub at
+https://github.com/jupyterlab/jupyterlab_apod.
+
+
+Build and install the extension for development
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Your new extension project has enough code in it to see it working in your
+JupyterLab. Run the following commands to install the initial project
+dependencies and install the extension into the JupyterLab environment.
 
 .. code:: bash
 
-    jlpm install
-    jupyter labextension install . --no-build
+    pip install -ve .
 
-After the install completes, open a second terminal. Run these commands
-to activate the ``jupyterlab-ext`` environment and to start a JupyterLab
-instance in watch mode so that it will keep up with our changes as we
-make them.
+The above command copies the frontend part of the extension into JupyterLab.
+We can either run it again every time we make a change to copy the change into
+JupyterLab, or on Linux or macOS, we can link the frontend extension part to
+our changes for a more natural development experience:
 
 .. code:: bash
 
-    conda activate jupyterlab-ext
-    jupyter lab --watch
+    jupyter labextension uninstall jupyterlab_apod
```

Comment:
We could instead run `jupyter labextension develop --overwrite .` and skip the `uninstall` step

Reply:
Yes, much better. Thanks!